eng/doc/fips/README.md

```diff
@@ -209,31 +215,30 @@ The following sections describe how to enable FIPS mode and the effect of the `G
 
 To set FIPS mode on Linux, use one of the following options. The first match in this list wins:
```

Comment:
This isn't really setting the mode anymore, and it might make sense to match the previous suggestion to keep calling it a preference:

```suggestion
To set FIPS mode preference on Linux, use one of the following options. The first match in this list wins:
```